src/Keboola/Console/Command/DeleteGoodDataProjects.php

```diff
@@ -0,0 +1,111 @@
+<?php
+namespace Keboola\Console\Command;
+
+use Console_CommandLine_Exception;
+use Exception;
+use Keboola\Csv\CsvFile;
+use Keboola\ManageApi\Client;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\ProgressBar;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DeleteGoodDataProjects extends Command
+{
+
+    const ARG_SOURCE_FILE = 'file';
+    const OPTION_FORCE = 'force';
+
+    protected function configure()
+    {
+        $this
+            ->setName('gooddata:delete-projects')
+            ->setDescription('Delete GoodData project in bulk')
+            ->addArgument(self::ARG_SOURCE_FILE, InputArgument::OPTIONAL, 'source CSV file', 'projects.csv')
```

Comment:
Nebylo by lepší spíš required?

Reply:
Proč? :)

Reply:
Jaká chápu, že když ho neuvedeš tak to zachytí ` [Exception] Source file does not exist: projects.csv`, ale co když ten default soubor bude existovat ale nebude to ten pravý nad jakým to chceš spustit?

Imho já bych to dal jako required, abych při spuštění explicitně uváděl soubor a v případě kdy nepochopím jak to funguje dostal:`[Exception] Not enough arguments (missing: "file")`